openapi3.yaml

```diff
@@ -61,12 +147,37 @@ components:
       properties:
         message:
           type: string
-    resource:
+    requestBodyForFilter:
+      type: object
+      required:
+        - arrayOfilterProduct
+      properties:
+        arrayOfilterProduct:
+          type: array
+          items:
+            $ref: '#/components/schemas/filterProduct'
+        limit:
+          type: integer
+          minimum: 0
+        skip:
+          type: integer
+          minimum: 0
+    filterProduct:
+              # expmple: {feild:'minZoom',oprator:'>',filterArg:'10'}
       type: object
       required:
-        - id
-        - name
-        - description
+        - feild
+        - oprator
+        - filterArg
+      properties:
+        feild:
+          type: string
+        operator:
+          type: string
+        filterArg:
+          type: string
```

Comment:
This needs to support [multiple types](https://swagger.io/docs/specification/data-models/data-types/).

Reply:
Operator is always string but according to your previous comment i add enum because i already know all the supported operators.  I changed the type of value to be multiple types.
```
  operator:
          type: string
          enum:
            - '<'
            - '>'
            - '<='
            - '>='
            - '='
            - '!='
            - 'st_contains'
            - 'st_within'
            - 'st_intersects'
        value:
          oneOf:
            - type: string
            - type: integer
```